### ISSUE TITLE
Reading plan?

### ISSUE THREAD

**Issue by mcepl** [zefanja/biblez-ng#18]
I think the Bible on the mobile phone is an ideal tool for the regular Bible reading. Would it be possible to somehow add a reading plan capability to biblez? This is a very broad RFE, I believe all details (e.g., should there be a depot of reading plans on the Internet, or is there one?, or should it be possible to import the plan from the SSD card?) could be discussed later. What do you think?

**Comment by zefanja** [comment]
Yes...we really should have this feature. I would like to add it, but my focus is to improve the performance of BibleZ, because it is not very fast. I've already started some work on this, but it is hard to find a good UI library that fits to Firefox OS. Mozilla doesn't make it very easy to use their building blocks. I like the current UI, but enyojs is to slow.

**Comment by mcepl** [comment]
Actually, I don't agree ... BibleZ (at least on Flame) works Well Enough™. Yes, it could be a bit faster, but certainly I wouldn't make it priority over other stuff (this or better av11n support).
